docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+services:
+  app:
+    build: .
+    container_name: chatbot_app
+    ports:
+      - "8000:8000"
+    depends_on:
+      - db
+    volumes:
+      - .:/app
+  db:
+    image: postgres:latest
+    container_name: chatbot_db
+    environment:
+      POSTGRES_USER: db_admin
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: chatbot_db
+    ports:
+      - "5433:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+volumes:
+  postgres_data:
```

Comment:
co do tego to : 

- uzywasz volumes .:/app i to jest ok na srodowisko DEV ale pamietaj ze jak bedizemy chcieli to deployowac w przyszlosci gdzies to moze nadpisac to pliki aplikacji
- zadbaj o to zeby w glownym katalogu byl .env i .env.dev np wtedy nie bedziesz musiala trzymac w ogole hasel w systemie
- rozwazylbym dodanie healthcheck bazy danych czy ona zyje :D 
- mozesz dodac restart: always dla obydwu serwisow w produkcji pozniejszej